pkg/collector/corechecks/cluster/orchestrator/orchestrator_processor.go

```diff
@@ -292,8 +344,61 @@ func processNodesList(nodesList []*corev1.Node, groupID int32, cfg *config.Orche
 		})
 	}
 
+	clusterMessage, err := extractClusterMessage(cfg, clusterID, client, groupID, nodeCount, kubeletVersions, podCap, podAllocatable, memoryAllocatable, memoryCap, cpuAllocatable, cpuCap)
+	if err != nil {
```

Comment:
In any error case we won't collect and just stop here.

Reply:
Or do we want to differentiate between nodes and logs error? But the only real way this will fail is if the apiserver is not reachable. We might want to consider still sending what we have collected so far (nodes) 🤔 . We shouldn't set clusters though as this will lead to incomplete/wrong data

Reply:
Implemented! But this comes with a cost of additional complexity.